pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -483,7 +483,7 @@ def make_package(args):
             join(res_dir, 'values/strings.xml'),
             **render_args)
 
-    if exists("custom_rules.tmpl.xml"):
+    if exists(join("templates", "custom_rules.tmpl.xml")):
```

Comment:
I'm not sure how this works if you fixed the first path in the `exists` condition, but not the `render` one which I would have expected to be the same :thinking:

Reply:
hi, because **os.path.exists** check on filesystem and not use **jinja2.Environment**, **render** it's a local function.

Reply:
@AndreMiras it works because of this line: https://github.com/kivy/python-for-android/blob/41b740eac3d4271eade67438cafa3ccaea9b98e9/pythonforandroid/bootstraps/common/build/build.py#L79 (which sets up the environment to be located inside `templates`, so jinja2 naturally looks in there)

Reply:
Thank you guys, clear!